src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -575,7 +575,9 @@ private void FinishAttachToMicrobe()
         //     !IsLoadedFromSave)
         //     ParentMicrobe.Colony.Master.Mass += Definition.Mass;
 
-        MakeCollisionShapes(ParentMicrobe!.Colony?.Master ?? ParentMicrobe);
+        // We don't need preview cells to be collidable (as it can lag the editor if the cell is massive).
+        if (!ParentMicrobe.IsForPreviewOnly)
+            MakeCollisionShapes(ParentMicrobe!.Colony?.Master ?? ParentMicrobe);
```

Comment:
I think the null suppression on ParentMicrobe is really unnecessary on this line now as the if will have crashed anyway before reaching this line.

Reply:
Sure, copied this without thought.